### PR TITLE
[entropy/doc] hw interface descriptions updated

### PR DESCRIPTION
**Pull request by mwbranstad**
A complete list of hardware interface updates have been made for entropy_src and csrng documention files.

Signed-off-by: Mark Branstad <mark.branstad@wdc.com>

### CODE REVIEW

File: hw/ip/entropy_src/doc/_index.md [hunk 1]
```diff
@@ -185,6 +191,23 @@ At any time, the `ENABLE` field can be cleared to halt the entropy generation (a
...
 {{< incGenFromIpDesc "../data/entropy_src.hjson" "hwcfg" >}}
 
+The table below lists other ENTROPY_SRC signals.
+
+Signal                         | Direction        | Type                        | Description
+-------------------------------|------------------|-----------------------------|---------------
+`otp_en_entropy_src_fw_read_i` | `input `         | `otp_en_t `                 | An efuse that will enable firmware to access final entropy through a register.
+`otp_en_entropy_src_fw_over_i` | `input `         | `otp_en_t `                 | An efuse that will enable firmware to access post-health check raw entropy through a register FIFO.
+`rng_fips_o`                   | `output`         | `logic `                    | Output that indicates to the RNG AST block that Entropy_src block is generating fips level entropy.
+`entropy_src_hw_if_i`          | `input`          | `entropy_src_hw_if_req_t`   | Seed request made to the Entropy_src module.
```
Comment (sriyerg) [inline hw/ip/entropy_src/doc/_index.md:201]: nit: inconsistent capitalization `Entropy_Src` v/s `Entropy_src` v/s `ENTROPY_SRC`

**Review by sriyerg — APPROVED** [review]
LGTM, thanks!

File: hw/ip/entropy_src/doc/_index.md (hunk 1 above)
Comment (mwbranstad) [reply hw/ip/entropy_src/doc/_index.md:201]: agreed - will fix.

**Review by senelson7 — APPROVED** [review]
LGTM

File: hw/ip/entropy_src/doc/_index.md (hunk 1 above)
Comment (sriyerg) [reply hw/ip/entropy_src/doc/_index.md:201]: @mwbranstad, will you fix this in a separate PR?

Comment (mwbranstad) [reply hw/ip/entropy_src/doc/_index.md:201]: @sriyerg weird, was sure this was fixed once. Will create a new PR.

**Pull request merged**